sale_operating_unit/models/sale_order.py

```diff
@@ -9,30 +9,59 @@
 class SaleOrder(models.Model):
     _inherit = "sale.order"
 
+    operating_unit_id = fields.Many2one(
+        comodel_name="operating.unit",
+        string="Operating Unit",
+        default=lambda self: self._default_operating_unit(),
+        readonly=False,
+        store=True,
+        precompute=True,
+        check_company=True,
+        compute="_compute_operating_unit_id",
```

Comment:
Why do you need a `default` method, since you have a `compute` one ?

Reply:
Code removed 👍